catalog/schematron/catalog-schematron-plugin/src/main/java/ddf/services/schematron/SchematronValidationService.java

```diff
@@ -266,6 +273,15 @@ private SchematronReport generateReport(String metadata, Templates validator)
         return report;
     }
 
+    /**
+     * Remove whitespace and literal newlines
+     * @param input
+     * @return
+     */
+    private String sanitize(final String input) {
+        return input.replaceAll("[\t \r\n]+", "");
```

Comment:
youdontwantyoursentencestolooklikethis

Change that second parameter to " " (<- space between the quotes)